### ISSUE TITLE
 data sample 

### ISSUE THREAD

**Issue by hpataci** [yash-007/NLP-with-Deep-Learning#1]
Hi there, amazing library and thank you so much. 

a short question.

" nlp-wth text summarization" folder you are doing multi-label data classification but you set 
y = binarizer.fit_transform(df['Tags']). In case we have multiple labels, it should come as a list, right?  otherwise, isnt this one a multi-class classification?

thank you so much in advance

**Comment by hpataci** [comment]
I just checked your code with BERT for multi-label classification too. I think what you mean is multi-class classification, not multi-label classification. It would be nicer to edit your code's title accordingly.

**Comment by theartificialguy** [comment]
> Hi there, amazing library and thank you so much.
> 
> a short question.
> 
> " nlp-wth text summarization" folder you are doing multi-label data classification but you set y = binarizer.fit_transform(df['Tags']). In case we have multiple labels, it should come as a list, right? otherwise, isnt this one a multi-class classification?
> 
> thank you so much in advance

Yes you are right, that was actually my bad... Thank you for pointing it out :) and sorry for late reply...
